FindOnPage.md

```diff
@@ -0,0 +1,478 @@
+    [propget] HRESULT FindTerm([out, retval] LPWSTR* value);
+    [propput] HRESULT FindTerm([in] LPCWSTR value);
+
+    // Gets or sets the direction of the find operation (forward or backward).
+    [propget] HRESULT FindDirection([out, retval] COREWEBVIEW2_FIND_DIRECTION* value); 
+    [propput] HRESULT FindDirection([in] COREWEBVIEW2_FIND_DIRECTION value);
+
+    // Gets or sets whether the find operation is case sensitive.
+    [propget] HRESULT IsCaseSensitive([out, retval] BOOL* value); 
+    [propput] HRESULT IsCaseSensitive([in] BOOL value);
+
+    // Gets or sets whether to only match whole words during the find operation.
+    [propget] HRESULT ShouldMatchWord([out, retval] BOOL* value); 
+    [propput] HRESULT ShouldMatchWord([in] BOOL value);
+
+    // Gets the state of whether all matches are highlighted.
+    // Returns TRUE if all matches are highlighted, FALSE otherwise.
+    [propget] HRESULT ShouldHighlightAllMatches([out, retval] BOOL* value); 
+    // Sets the state to either highlight all matches or not.
+    [propput] HRESULT ShouldHighlightAllMatches([in] BOOL value);
+ 
+    // Checks if a custom user interface is desired by end developer.
+    // If TRUE, the default Find bar is not displayed.
+    // Returns TRUE if using a custom UI, FALSE if using the default.
+    [propget] HRESULT SuppressDefaultDialog([out, retval] BOOL* value); 
+    // Sets whether to use a custom UI for the Find operation.
+    [propput] HRESULT SuppressDefaultDialog([in] BOOL value);
+
+    // Gets the index of the currently active match.
+    // If there's no active find session but an attempt is made to get the active match index:
+    // The function will return -1.
+    // Returns The active match index.
+    [propget] HRESULT ActiveMatchIndex([out, retval] LONG* value); 
+
+    // Gets the total count of matches found in the current document based on the last find criteria.
+    // Returns the total count of matches.
+    [propget] HRESULT MatchesCount([out, retval] LONG* value);
+
+}
+
+/// Handles the event that's fired when the match count changes.
+[uuid(623EFBFB-A19E-43C4-B309-D578511D24AB), object, pointer_default(unique)]
+interface ICoreWebView2FindMatchCountChangedEventHandler : IUnknown {
+    /// Parameter is the match count.
+    HRESULT Invoke(LONG matchesCount);
+}
+
+/// Handles the event that's fired when the active match index changes.
+[uuid(623EFBF9-A19E-43C4-B309-D578511D24A9), object, pointer_default(unique)]
+interface ICoreWebView2FindActiveMatchIndexChangedEventHandler : IUnknown {
+    /// Parameter is the active match index.
+    HRESULT Invoke(LONG activeMatchIndex);
+}
+/// Handles the event that's fired when the find operation completes.
+[uuid(2604789D-9553-4246-8E21-B9C74EFAD04F), object, pointer_default(unique)]
+interface ICoreWebView2FindOperationCompletedHandler : IUnknown {
+    /// Provides the event args when the find operation completes.
+    HRESULT Invoke(HRESULT errorCode, BOOL status);
```

Comment:
What does `BOOL status` represent? That should be described in the documentation for the corresponding method.